test/DiceTests.cpp

```diff
@@ -0,0 +1,10 @@
+#include <gtest/gtest.h>
+#include "Dice.hpp"
+
+struct DiceTest : public ::testing::Test
+{
+    Dice firstDice;
```

Comment:
There are no tests for throwDice(), getValue() and operator+(). You should always commit only tested code.